lib/csvlint.rb

```diff
@@ -8,10 +8,18 @@
 require 'active_support/core_ext/time/conversions'
 require 'mime/types'
 require 'open_uri_redirections'
+require 'uri_template'
 
 require 'csvlint/error_message'
 require 'csvlint/error_collector'
 require 'csvlint/validate'
 require 'csvlint/wrapped_io'
 require 'csvlint/field'
+require 'csvlint/csvw_metadata_error'
+require 'csvlint/csvw_number_format'
```

Comment:
Might be nice to put all the csv on the web folders in a seperate folder, rather than with the `csvw` prefix, and also namespace them, so `CsvwColumn` for example would be in `lib/csvlint/csvw/column` and namespaced as `Csvlint::Csvw::Column`. Just keeps things neater :smile: